### DISCUSSION TITLE
FleetDM - Asset Discovery/Management

### DISCUSSION THREAD

**Discussion by ow111** [Security-Onion-Solutions/securityonion#4201] [Ideas] [answered]
Recently, I have seen that the latest version of FleetDM includes a software inventory for each host on their details page,  which can be super useful for quickly finding out some info on a host. 
Wondering if FleetDM could be expanded upon, such that ndiffs' of nmap scans (as an example) could be imported into the GUI to add functionality of asset discovery on a given subnet or otherwise.

Is there extra tooling that SO are looking at for Asset Discovery/Management ? 

https://github.com/fleetdm/fleet/releases
Add support for software inventory, introducing a list of installed software items on each host's respective Host details page. This feature is flagged off by default (for now)

**Comment by defensivedepth** [comment, marked as answer]
Sounds like a good suggestion! Best place for it is over in the FleetDM github as a Feature Request.

As for extra tooling for SO for Discovery/Management - Out of the box, SO 2 supports Osquery & Wazuh agents. There are no plans at this time to support a tool that is specific to Asset Discovery/Management.

Thanks!
